source/mod_time.f90

```diff
@@ -25,28 +26,39 @@ module mod_time
   integer, parameter :: time_afterPostTrack      = 9
   integer, parameter :: time_afterPostProcessing = 10
   integer, parameter :: time_afterFMA            = 11
-  integer, parameter :: time_afterHASH           = 12
-  integer, parameter :: time_afterZIPF           = 13
-  integer, parameter :: time_beforeExit          = 14
+  integer, parameter :: time_afterAllPostPR      = 12
+  integer, parameter :: time_afterHASH           = 13
+  integer, parameter :: time_afterZIPF           = 14
+  integer, parameter :: time_beforeExit          = 15
 
-  real(kind=fPrec), public,  save :: time_timeZero = 0.0
-  real(kind=fPrec), public,  save :: time_timeRecord(time_beforeExit)
+  real(kind=fPrec), public,  save :: time_lastTick = 0.0
+  real(kind=fPrec), private, save :: time_timeZero = 0.0
+  real(kind=fPrec), private, save :: time_timeRecord(time_beforeExit)
 
   ! Constants for accumulated time
-  integer, parameter :: time_nClocks   = 4
-  integer, parameter :: time_clockDUMP = 1
-  integer, parameter :: time_clockCOLL = 2
-  integer, parameter :: time_clockSCAT = 3
-  integer, parameter :: time_clockHDF5 = 4
-
-  real(kind=fPrec), public,  save :: time_clockStart(time_nClocks)
-  real(kind=fPrec), public,  save :: time_clockTotal(time_nClocks)
-  integer,          public,  save :: time_clockCount(time_nClocks)
-  integer,          public,  save :: time_clockStops(time_nClocks)
+  integer, parameter :: time_nClocks   = 5
+  integer, parameter :: time_clockCR   = 1
+  integer, parameter :: time_clockDUMP = 2
+  integer, parameter :: time_clockCOLL = 3
+  integer, parameter :: time_clockSCAT = 4
+  integer, parameter :: time_clockHDF5 = 5
```

Comment:
same comment here as before. A kind of python dictionary would make addition of new modules to be timed easier...